languages/elixir/exercises/concept/basic-strings/.docs/introduction.md

```diff
@@ -0,0 +1,46 @@
+Strings in Elixir are delimited by double quotes, and they are encoded in UTF-8:
+
+```elixir
+iex> "Hi! ¡Hola! Cześć! Привет!"
+"Hi! ¡Hola! Cześć! Привет!"
+```
+
+Strings can be concatenated using the `<>/2` operator:
+
+```elixir
+iex> "Welcome to" <> " " <> "New York"
+"Welcome to New York"
+```
+
+Strings in Elixir support interpolation using the `#{}` syntax:
+
+```elixir
+iex> "6 * 7 = #{6 * 7}"
+"6 * 7 = 42"
+```
+
+Elixir provides many functions for working with strings in the _String module_.
+
+```elixir
+iex> String.downcase("HELLO")
+"hello"
+
+iex> String.last("12345")
+"5"
+```
```

Comment:
The example of `String` module calls don't introduce new syntax. The `introduction.md` file should only have code examples for new syntax (see [the documentation](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsinstructionsmd)). Maybe highlight the `Strings` module using backticks too to have it be clear that it is referencing code?

```suggestion
Elixir provides many functions for working with strings in the `String` module.
```

Reply:
Done!